consensus/ConsensusReporter.py

```diff
@@ -108,10 +109,10 @@ def process_consensus(self):
         print('Generating reports')
         self.count_classifications()
         progress.update(1)
-        self.report.write('chromosome,position,ref,alt,gene,transcript,c dna')
+        self.report.write('chromosome\tposition\tref\talt\tgene\ttranscript\tc_dna')
         for lab in self.labs:
-            self.report.write(',' + lab)
-        self.report.write('\n')
+            self.report.write('\t' + lab)
+        self.report.write(os.linesep)
```

Comment:
I prefer the previous solution where the output is equal regardless of the operating system that it is run on.